File: DataStructures/Array/rotateArray.java
//with java you write class files 

class RotateArray {

    public void leftRotate(int arr[], int times)
    {
        for(int i = 0; i < times; i++)
        {
            leftRotateByOne(arr, length);
        }
    }

    public void leftRotateByOne(int arr[])
    {
        int temp = arr[0];
        for(int i = 0; i < arr.length; i++)
        {
            arr[i] = arr[i + 1];
        }

        arr[length - 1] = temp;
    }

    public void printArray(int arr[])
    {
        for(int i = 0; i < arr.length; i++)
        {
            System.out.print(arr[i] + " ");
        }
    }

    public static void main(String[] args)
    {
        RotateArray rotate = new RotateArray(); // declare new class instance
        int arr[] = {1, 2, 3, 4, 5, 6, 7 };
        rotate.leftRotate(arr, 2, 7);
        rotate.printArray(arr, 7);
    }
}

File: DataStructures/Array/rotateArray.cs
using System;

class rotateArray {
    static void leftRotate(int[] arr, int times, int length)
    {
        for(int i = 0; i < times; i++)
        {
            leftRotateByOne(arr, n);
        }
    }

    static void leftRotateByOne(int[] arr, int length)
    {
        int temp = arr[0];
        for(int i = 0; i < length - 1; i++)
        {
            arr[i] = arr[i+1];
        }
        arr[length-1] = temp;
    }

    static void printArray(int[] arr, int length)
    {
        for(int i = 0; i < length; i++)
        {
            Console.Write(arr[i] + " ");
        }
    }

    public static void Main()
    {
        int[] arr = {1, 2, 3, 4, 5, 6, 7};
        leftRotate(arr, 2, 7);
        printArray(arr, 7);
    }
}